griptape/tasks/base_task.py

```diff
@@ -168,3 +174,15 @@ def full_context(self) -> dict[str, Any]:
             return structure_context
         else:
             return {}
+
+    def _add_to_structure(self, task: Optional[BaseTask]):
+        if task is not None:
+            if self.structure is not None and task.structure is not None and self.structure != task.structure:
+                raise ValueError("Task is already associated with a different structure.")
+            elif self.structure:
```

Comment:
Use `is not None` check

Reply:
is there a linting rule that can be added for this?